sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcReadSchemaTransformProvider.java

```diff
@@ -241,7 +368,7 @@ public abstract static class Builder {
 
       public abstract Builder setConnectionInitSql(List<String> value);
 
-      public abstract Builder setFetchSize(Short value);
+      public abstract Builder setFetchSize(Integer value);
```

Comment:
Why did we change the typing here?

Reply:
The fetchSize on JdbcIO accepts Integer values, so this provider was cutting out a large range of acceptable values